translations/en-us.yaml

```diff
@@ -1209,6 +1209,7 @@ globalRegistryPage:
     adminPassword:
       label: Admin Password
       placeholder: Harbor Admin password
+      helpText: "If you re-use an existing persistent volume claim for Database component, the admin password will not take effect."
```

Comment:
The password will take effect, if the existing PVC bounds to a fresh PV.
See case 2 in the comment: https://github.com/rancher/rancher/issues/20797#issuecomment-501515382

Reply:
How about `If you are using the existing PVC which bounds to a used PV, the password will not take effect`

Reply:
I prefer `If you are using the existing PVC with initialized Harbor data, the password will not take effect`